taskcluster/docker/csharp/Dockerfile

```diff
@@ -0,0 +1,117 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster
+
+LABEL maintainer="aplacitelli@mozilla.com"
+
+RUN echo "trigger rebuild"
+
+# Add worker user for TaskCluster.
+
+RUN mkdir /builds && \
+    useradd -d /builds/worker -s /bin/bash -m worker && \
+    chown worker:worker /builds/worker && \
+    mkdir /builds/worker/artifacts && \
+    chown worker:worker /builds/worker/artifacts
+
+WORKDIR /builds/worker/
+
+# Configuration
+ENV DOTNET_CLI_TELEMETRY_OPTOUT true
+
+# Set up the language variables to avoid problems (we run locale-gen later).
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# Do not use fancy output on taskcluster
+ENV TERM dumb
+
+# Used to detect in scripts whether we are running on taskcluster
+ENV CI_TASKCLUSTER true
+
+ENV \
+    # Some APT packages like 'tzdata' wait for user input on install by default.
+    # https://stackoverflow.com/questions/44331836/apt-get-install-tzdata-noninteractive
+    DEBIAN_FRONTEND=noninteractive
+
+# System.
+RUN apt-get update -qq \
+    && apt-get install -qy --no-install-recommends \
+        # To fetch Glean.
+        git \
```

Comment:
Where does the clone happen? Is it automatically performed by TC? Does it mean I can get rid of this dependency?

Reply:
afaik TC inserts that automatically into jobs, meaning we need to keep `git` around.